cmd/zrepl/zrepl.c

```diff
@@ -272,10 +273,6 @@ uzfs_zvol_worker(void *arg)
 		ZREPL_ERRLOG("Zvol op_code :%d failed with "
 		    "error: %d\n", hdr->opcode, errno);
 		hdr->status = ZVOL_OP_STATUS_FAILED;
-	} else {
-		ZREPL_LOG("Zvol io_seq:%ld op_code :%d passed\n",
-		    hdr->io_seq, hdr->opcode);
-		hdr->status = ZVOL_OP_STATUS_OK;
```

Comment:
it seems - this is required.

Reply:
Opps, It seems this removed along with removing log message.